examples/simple-platform/run-example.md

```diff
@@ -0,0 +1,144 @@
+# Key pair for the platform node
+export PUBLIC_KEY=""
+export PRIVATE_KEY=""
+# Peer list override
+export PORT=5025
+export CONNECT_TO_PEERS="localhost:8888"
+```
+1. Copy the `GITHUB_CLIENT_ID`, `GITHUB_CLIENT_SECRET`, and 
+`GITHUB_REDIRECT_URL` 
+values from the GitHub OAuth app you just created. 
+1. The `PUBLIC_KEY` and `PRIVATE_KEY` values come from 
+`platform-deploy-keys.json`. 
+
+### Set up environment variables for `web-auth-endpoint`
+
+1. Create a `/secrets/web-auth-endpoint.sh` file with the following content. 
+```bash
+#!/bin/bash
+# Secrets for Accounts App GitHub OAuth chain
+export GITHUB_CLIENT_ID=""
+export GITHUB_CLIENT_SECRET=""
+# Key pair for the web auth node
+export PUBLIC_KEY=""
+export PRIVATE_KEY=""
+# Peer list override
+export PORT=8888
+export CONNECT_TO_PEERS="localhost:5025"
+```
+1. The `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` are the same as in your 
+`secrets/platform-deploy.sh` file.
+1. The `PUBLIC_KEY` and `PRIVATE_KEY` values come from 
+`web-auth-endpoint-key.json`.
+
+## Spinning up the platform 
+
+### Build and start the `platform-deploy` node
+
+1. Delete the existing manifest file, 
+`packages/platform-deploy/platform/interbit.manifest.json`. This file must be 
+deleted when new Interbit key pairs are generated (as we did in our setup). If 
+there is a pre-existing manifest, the genesis blocks will not be overwritten 
+and our new keys will not work. Note that when we generate a new manifest, we 
+create new genesis blocks which results in new chain IDs (which are hashes of 
+the genesis blocks). 
+
+1. Open a terminal shell and run the following commands from the `interbit`
+repo's root:
+```
+npm i
+. platform-deploy.sh
+cd packages/platform-deploy
+npm start
+```
+
+Our package script runs a post-install step which builds all the modules in 
+the Interbit repo. The build step for `platform-deploy` creates a new manifest 
+file with the newly generated keys and hashed covenant files. The build step 
+also updates the chain IDs for the other modules, including `app-account` and 
+`interbit-template`. 
+
+The `start` command spins up the `platform-deploy` server node, which listens 
+for incoming connections from the `web-auth-endpoint` server node and the 
+`app-account` and `interbit-template` browser nodes. 
+
+Leave this terminal shell open. 
+
+### Start the `web-auth-endpoint` hypervisor on a node
+
+Open a new terminal shell and run the following commands from the `interbit` 
+repo's root: 
+```
+. /secrets/web-auth-endpoint.sh
+cd packages/web-auth-endpoint
+npm start
+```
+Again, leave this shell open. 
+
+### Start the app-account and interbit-template apps
+
+In a third terminal, run the following from the `interbit` repo root
+```
+cd packages/app-account
+npm start
+```
+
+And, in a fourth terminal 
+```
+cd packages/interbit-template
+npm start
+```
+
+We've now started the browser nodes for the accounts and template apps. Your 
```

Comment:
~browser nodes~ webpack dev servers which will serve our react apps. (which will start browser nodes when they are opened in a browser)